src/test/resources/cluster.xml

```diff
@@ -90,15 +90,7 @@
     <queue-capacity>0</queue-capacity>
   </executor-service>
 
-  <multimap name="__vertx.subs">
-
-    <!--
-        Number of backups. If 1 is set as the backup-count for example,
-        then all entries of the map will be copied to another JVM for
-        fail-safety. 0 means no backup.
-    -->
-    <backup-count>1</backup-count>
-  </multimap>
+  <replicatedmap name="__vertx.subs"/>
```

Comment:
why is this removed ?

Reply:
The subs will now be stored in a replicated map instead of a multimap. At the expense of a bigger memory footprint, it makes storage reliable and relieves the implementation from publishing subs again when topology changes.